src/test/java/edu/kis/powp/jobs2d/events/SelectTestImmutableCommandBuilderTest.java

```diff
@@ -0,0 +1,20 @@
+package edu.kis.powp.jobs2d.events;
+
+import edu.kis.powp.jobs2d.command.CompoundCommand;
+import edu.kis.powp.jobs2d.command.TaskImmutableCommandBuilder;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class SelectTestImmutableCommandBuilderTest implements ActionListener {
+
+    @Override
+    public void actionPerformed(ActionEvent e) {
+        CompoundCommand compoundCommand = new TaskImmutableCommandBuilder(5).setName("Name").addCommandsList("clone").build();
```

Comment:
I do not see the point in this test. No real command (e.g., shape) is actually created. Nothing is shown on the panel or no command loaded to CommandManager.